src/lib/pages/backoffice/Document/DocumentDetails/DocumentPurchaseOrders/DocumentPurchaseOrders.js

```diff
@@ -0,0 +1,112 @@
+
+import { dateFormatter } from '@api/date';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withCancel } from '@api/utils';
+import { orderApi } from '@api/acquisition';
+import { ResultsTable } from '@components/ResultsTable/ResultsTable';
+import { Loader } from '@components/Loader';
+import { Link } from 'react-router-dom';
+import { AcquisitionRoutes } from '@routes/urls';
+
+export default class DocumentPurchaseOrders extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      error: false,
+      orders: {},
+      isLoading: false
+    };
+    const {
+      documentDetails: { pid },
+    } = this.props;
+    this.fetchPurchaseOrders(pid)
+  }
+
+  componentWillUnmount() {
+    this.cancellablePurchaseOrders && this.cancellablePurchaseOrders.cancel();
+  }
+
+  fetchPurchaseOrders = async (pid) => {
+    try {
+      this.cancellablePurchaseOrders = withCancel(
+        orderApi.list(orderApi.query().withDocPid(pid).qs())
+      );
+      const response = await this.cancellablePurchaseOrders.promise;
+      this.setState({ orders: response.data});
+    } catch (error) {
+      if (error !== 'UNMOUNTED') {
+        error = true;
+      }
+    }
+  };
+
+  viewDetails = ({ row }) => {
+    return (
+      <Link
+        to={AcquisitionRoutes.orderDetailsFor(row.id)}
+      >
+        {row.id}
+      </Link>
+    );
+  };
+
+
+  renderTable(data){
+    const columns = [
+      { title: 'Order', formatter: this.viewDetails },
+      {
+        title: 'Status',
+        field: 'metadata.status',
+      },
+      {
+        title: 'Created date',
+        field: 'created',
+        formatter: dateFormatter,
+      },
+      {
+        title: 'Updated date',
```

Comment:
is the modification date important from the librarian point of view ? maybe there is other property of the order which they would like to see here ?